ingolfur/index.html

```diff
@@ -0,0 +1,181 @@
+    <meta name="twitter:description" content="100% FREE Professional Portfolio Templates - Ingólfur Template" />
+
+    <!-- mobile specific metas -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- CSS -->
+    <link rel="stylesheet" href="css/base.css">
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
+
+    <!-- favicons -->
+    <link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
+    <link rel="manifest" href="site.webmanifest">
+
+</head>
+
+<body id="top">
+
+    <!-- preloader -->
+    <div id="preloader">
+        <div class="spinner" id="spinner"></div>
+    </div>
+
+    <!-- header -->
+    <header class="s-header">
+
+        <div class="header-logo">
+            <a href="index.html">
+                <svg xmlns="http://www.w3.org/2000/svg" shape-rendering="geometricPrecision" text-rendering="geometricPrecision" image-rendering="optimizeQuality" fill-rule="evenodd" clip-rule="evenodd" viewBox="0 0 493 511.69"><path fill-rule="nonzero" d="M6.99 0h314.2v154.84H0V0h6.99zm345.75 356.85H493v154.84H345.75V356.85h6.99zm126.29 13.97H359.72v126.9h119.31v-126.9zM6.99 356.85h314.2v154.84H0V356.85h6.99zm300.23 13.97H13.97v126.9h293.25v-126.9zM133.28 192.39H13.97V319.3h119.31V192.39zM6.99 178.42h140.26v154.85H0V178.42h6.99zm472.04 13.97H185.78V319.3h293.25V192.39zM178.8 178.42H493v154.85H171.82V178.42h6.98zM352.74 0H493v154.84H345.75V0h6.99zm126.29 13.97H359.72v126.9h119.31V13.97zm-171.81 0H13.97v126.9h293.25V13.97z"/></svg>
+            </a>
+        </div>
+
+        <div class="header-content">
+    
+            <nav class="row header-nav-wrap">
+                <ul class="header-nav">
+                    <li><a class="smoothscroll active-link" href="#hero" title="Intro">Home</a></li>
+                    <li><a class="smoothscroll" href="#projects" title="Works">Projects</a></li>
+                    <li><a href="mailto:#0" title="Contact us">Contact Me</a></li>
+                </ul>
+            </nav>
+
+            <a href="#0" class="btn btn--stroke btn--small">Download CV</a>
+
+        </div>
+
+        <a class="header-menu-toggle" href="#0"><span>Menu</span></a>
+
+    </header> 
+
+    <!-- hero -->
+    <section id="hero" class="s-hero target-section" data-parallax="scroll" data-image-src="images/hero-bg.jpg" data-natural-width=3000 data-natural-height=2000 data-position-y=center>
+
+        <div class="row hero-content">
+
+            <div class="column large-full">
+
+                <h1>
+                Hi there, I'm Jane Smith, <br>
+                a creative web designer <br>
+                and frontend developer <br>
+                located in Philippines.
```

Comment:
Replace "located" with either "based in the Philippines" or "form the Philippines" Thanks.

Reply:
Hello sir @jofftiquez, I have updated the text in the hero section. Please refer to the image below. Thank you!

![image](https://user-images.githubusercontent.com/26820513/229131289-b918a4ca-b3b2-49d8-810b-e09c77ccf185.png)